README.md

```diff
@@ -2,9 +2,6 @@
 
 RBS::Inline allows embedding RBS type declarations into Ruby code as comments. You can declare types, write the implementation, and verifies they are consistent without leaving the editor opening the Ruby code.
 
-> [!IMPORTANT]
-> The syntax is experimental. We are still seeking the best syntax with your feedbacks.
-
 > [!IMPORTANT]
 > This gem is a prototype for testing. We plan to merge this feature to rbs-gem and deprecate rbs-inline gem after that.
 
```

Comment:
I think the NOTE below is also able to be removed because Steep-1.8 was already released.

Reply:
* #142